src/main/java/edu/oregonstate/cs361/battleship/BattleshipModelNormal.java

```diff
@@ -44,19 +43,15 @@ public BattleshipModelNormal() {
         computerHits = new ArrayList<Point>();
         computerMisses = new ArrayList<Point>();
 
-        playerShips = new ArrayList<Ship>();
-        playerShips.add(playerAircraftCarrier);
-        playerShips.add(playerBattleship);
-        playerShips.add(playerCruiser);
-        playerShips.add(playerDestroyer);
-        playerShips.add(playerSubmarine);
-
-        computerShips = new ArrayList<Ship>();
-        computerShips.add(computerAircraftCarrier);
-        computerShips.add(computerBattleship);
-        computerShips.add(computerCruiser);
-        computerShips.add(computerDestroyer);
-        computerShips.add(computerSubmarine);
+        playerArray[1] = playerBattleship;
+        playerArray[2] = playerSubmarine;
+        playerArray[3] = playerCruiser;
+        playerArray[4] = playerDestroyer;
+
+        computerArray[1] = computerBattleship;
+        computerArray[2] = computerSubmarine;
+        computerArray[3] = computerCruiser;
+        computerArray[4] = computerDestroyer;
```

Comment:
With Lists you can use functions like .contains() or .add() and let the list sort out things. Arrays mean we have to remember the ship order and hardcode the indexes. Arrays have less overhead but the potential to mix up the order is there. This will do tho.